litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowComparisonTable/TableHeader.tsx

```diff
@@ -139,7 +139,7 @@ const TableHeader: React.FC<TableHeaderProps> = ({
         </TableCell>
         <TableCell className={classes.headSpacing}>
           <div className={classes.nameContent}>
-            <b>Regularity</b>&nbsp;
+            &nbsp;&nbsp;<b>Regularity</b>&nbsp;
```

Comment:
keep this in translation

Reply:
These table headers are not having translations for any table @oumkale we can discuss this change if it is needed for all the table, changing it now for just this table will give conflicts later in translation file for all tables.